Source/Core/VideoBackends/D3D/TextureCache.cpp

```diff
@@ -177,13 +177,11 @@ void TextureCache::TCacheEntry::FromRenderTarget(u32 dstAddr, unsigned int dstFo
 		u8* dst = Memory::GetPointer(dstAddr);
 		size_t encoded_size = g_encoder->Encode(dst, dstFormat, srcFormat, srcRect, isIntensity, scaleByHalf);
 
-		u64 hash = GetHash64(dst, (int)encoded_size, g_ActiveConfig.iSafeTextureCache_ColorSamples);
-
 		size_in_bytes = (u32)encoded_size;
 
 		TextureCache::MakeRangeDynamic(dstAddr, (u32)encoded_size);
 
-		this->hash = hash;
+		this->hash = GetHash64(dst, (int)encoded_size, g_ActiveConfig.iSafeTextureCache_ColorSamples);
```

Comment:
paging @degasus or @neobrain , is it OK to place the hash calc after MakeRangeDynamic? It looked OK to me, but I'm not entirely sure :)